OrbitGl/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 cmake_minimum_required(VERSION 3.15)
 
 project(OrbitGl CXX)
-add_library(OrbitGl STATIC)
+add_library(OrbitGl STATIC TracepointsDataView.h TracepointsDataView.cpp)
```

Comment:
Move to the "target_sources" below (ss it is done for the other *DataView.{cpp,h} files).